ledger/ledger-configuration/src/main/scala/com/daml/ledger/configuration/Configuration.scala

```diff
@@ -21,13 +21,26 @@ final case class Configuration(
 )
 
 object Configuration {
+  val NoGeneration: Long = 0
+  val StartingGeneration: Long = 1
```

Comment:
Do we need them public?

Reply:
Yes, they're not used here, only elsewhere. Not much happens inside this package.

Reply:
I was thinking about package restriction; if they need to be fully public, maybe it's worth making them a bit more opaque?

Reply:
Other ledger operators may need to access `StartingGeneration` (for example, if they don't want to use `reasonableInitialConfiguration`). I don't see any particular harm in keeping that one public.

I just realized `NoGeneration` is not actually used except in a test case now, so I think I'll just delete that.